tests/ping.go

```diff
@@ -80,6 +80,33 @@ func PingFromVMConsole(vmi *v1.VirtualMachineInstance, ipAddr string, args ...st
 	return nil
 }
 
+func PingHTTPFromConsole(vmi *v1.VirtualMachineInstance, host string, port int) error {
+	const maxCommandTimeout = 10 * time.Second
+
+	err := vmiConsoleExpectBatch(vmi, []expect.Batcher{
+		&expect.BSnd{S: "\n"},
+		&expect.BExp{R: PromptExpression},
+		&expect.BSnd{S: fmt.Sprintf("curl --silent %s:%d > /dev/null\n", host, port)},
```

Comment:
We may need to comment about this dependency (`curl`) in the doctext. What do you think?